lib/apia/open_api/specification.rb

```diff
@@ -14,14 +14,17 @@ class Specification
 
       OPEN_API_VERSION = "3.0.0" # The Ruby client generator currently only supports v3.0.0 https://openapi-generator.tech/
 
-      def initialize(api, base_url, name, info = {}, external_docs = {})
+      def initialize(api, base_url, name, additions)
```

Comment:
I think it could be `additions = {}` in the method definition. 

It just means someone can do `Specification.new(api, "url", "name")` without having to specify any additions at all if they didn't want to.

Which might mean also those changes to `MockApi, base_url_option, name_option` might not be required.

Reply:
On that last point 
> Which might mean also those changes to MockApi, base_url_option, name_option might not be required.

I think I'm defaulting the values to `{}` for each in the rack setup so they're required. Interested in better ways of handling that :pray:

Reply:
OK yeah I see now, they're basically hard-coded as part of the rack setup. I think it's fine like that. It's not like we actually have any default values for those additional options. If it started to get more complex we could maybe extract it out into a class of its own, but it's definitely not worth it at this point. A simple hash is plenty good enough.